app/views/coronavirus_landing_page/show.html.erb

```diff
@@ -1,3 +1,14 @@
+<% ga4_link_attributes = {
```

Comment:
Is it worth having these attributes defined as a variable when they're only used once, and its seventy lines away? I think it'd be fine to keep together with the rest of the code for this.

Reply:
Won't this work?

```Ruby
<%= content_tag :section, 
  class: "covid__topic-wrapper", 
  data: { 
    module: "ga4-link-tracker", 
    ga4_link: ga4_link_attributes,
    ga4_track_links_only: "",
    ga4_link: {
      event_name: "navigation",
      type: "subscribe",
      index: {
        index_link: 1
      },
      index_total: 1,
      section: "Footer"
    }
  }
} do
```

Reply:
D'oh! Of course. My brain was not working at all. I've made that change now, thanks for spotting it 👍